src/server/bridges/AppBridges.ts

```diff
@@ -43,8 +42,7 @@ export type Bridge =
     | SchedulerBridge
     | VideoConferenceBridge
     | OAuthAppsBridge
-    | ModerationBridge
-    | RoleBridge;
+    | ModerationBridge;
```

Comment:
Why are we removing the `Roles` bridge? Maybe this feature branch was not synced with the latest `alpha` 🤔 If so, could you please update this branch?

Reply:
ouch! Reverting the revert...